test/basic.html

```diff
@@ -206,6 +206,211 @@
         });
       });
 
+      suite('accessibility', function() {
+        var LEFT = 37;
+        var RIGHT = 39;
+        var tabs;
+
+        setup(function () {
+          tabs = fixture('basic');
+          Polymer.dom.flush();
+        });
+
+        test('paper-tabs has role tablist', function() {
+          assert.equal(tabs.getAttribute('role'), 'tablist');
+        });
+
+        test('paper-tab has role tab', function() {
+          tabs.items.forEach(function(tab) {
+            assert.equal(tab.getAttribute('role'), 'tab');
+          });
+        });
+
+        test('tabs are selected when moved to using arrow keys', function(done) {
+          ensureDocumentHasFocus();
+          Polymer.Base.async(function() {
+            tabs.select(0);
+            MockInteractions.pressAndReleaseKeyOn(tabs.selectedItem, RIGHT);
+            Polymer.Base.async(function() {
+              assert.equal(tabs.selected, 1);
+
+              MockInteractions.pressAndReleaseKeyOn(tabs.selectedItem, RIGHT);
+              Polymer.Base.async(function() {
+                assert.equal(tabs.selected, 2);
+
+                MockInteractions.pressAndReleaseKeyOn(tabs.selectedItem, LEFT);
+                Polymer.Base.async(function() {
+                  assert.equal(tabs.selected, 1);
+                  done();
+                }, 1000);
```

Comment:
`setTimeout`'s delay seems to be more of a suggestion. I've noticed I can set this about as low as 100ms without it getting too flaky. Fun times ahead: https://gist.github.com/bicknellr/0a81f12860ad9a8f4930. Also, try changing `spool(100, 5)` to `spool(1000, 1)`; in most browsers (Safari being the only exception I've found), ordering isn't even guaranteed when the calls are made from the same execution context. :( `setTimeout` sucks.